### DISCUSSION TITLE
In detail window of one line/marker, show it's type

### DISCUSSION THREAD

**Discussion by georg-d** [FacilMap/facilmap#241] [Ideas]
In both maps I use, there are several types defined, e.g. "done" and "todo". When clicking a marker or line, it's details are shown (coordinates, note etc):
![grafik](https://github.com/FacilMap/facilmap/assets/45993293/3c27dc95-7d4a-4cc2-8daa-355bbc8d25a7)
Screenshot from https://facilmap.org/GeorgsTestA#16/47.9755/7.8661 (replace A with RW and W for different access rights)

IMHO it would be helpful to also show the name of the type that line/marker is contained in (pink text in screenshot), because for unfamilar users, 

- with read+write access, it is difficult to anticipate they must click on "edit data" then "change type" to see the current type's name.
- with read only access, AFAIK it is completely impossible to see the type as well as a list of available types (hence, they can also not filter and learn via that way which line/marker is in which type, except legend is active)
- with any access, the visual appearance of lines does usually _not clearly_ transport the meaning, while for markers, it _may be possible_ with if a well fitting icon exists

What do you think? 🙂

**Comment by cdauth** [comment]
This is a good idea, feel free to create an issue from it.

In my maps what I do is define fixed styles for the different types and show them in the legend. This way it is usually clear what type I'm looking at.

**Reply by georg-d** [reply]
I also use fixed styles, but how do unfamiliar users know what a pink line means in a map _without legend_ (i.e. default)? 🤷‍♂️

**Comment by cdauth** [comment]
Tracked as #243.
